README.md

```diff
@@ -271,21 +267,22 @@ The supported individual operations are:
 let compile = table.Template.PrecomputeConditionalExpr
 let doesntExistCondition = compile <@ fun t -> NOT_EXISTS t.Value @>
 let existsCondition = compile <@ fun t -> EXISTS t.Value @>
-
 let key = TableKey.Combined(hashKey, rangeKey)
-let requests = [
-    TransactWrite.Check  (key, doesntExistCondition)
-    TransactWrite.Put    (item2, None)
-    TransactWrite.Put    (item3, Some existsCondition)
-    TransactWrite.Delete (table.Template.ExtractKey item5, None) ]
-do! table.TransactWriteItems requests
+
+let transaction = Transaction()
+
+transaction.Check(table, key, doesntExistCondition)
```

Comment:
the current API is returning `this`, which means this needs lots of ugly `ignore`s